client/src/components/StreamDetail.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const StreamDetail = ({totalViews, watchingNow, streamTitle, gamePoster, category}) => (
+    <div id='stream-detail-section'>
+        <div id='header-container'>
+            <div style={{ 'width': '100%', 'display': 'grid' }} >
+                <div style={{ 'width': '100%' }}>
+                    <h2 id='stream-title'>{streamTitle}</h2>
+                </div>
+                <div id='game-details'>Category: {category}</div>
+            </div>
+            <div id='current-and-total-views'
+            >
+                <div id='watching-now-container'>
+                    <div id='watching-now'>{watchingNow} </div>
+                </div>
+                <div id='total-views-container'>
+                    <div id='total-views'>
+                        {totalViews}
```

Comment:
You may eventtually find that you want to factor these out into react components (especially if they ever need things like mine did -- tooltips that pop up when onHover).

Reply:
I'm curious if I have the time at the moment. My css flow look suitable for css module if I can configure correctly. If not then it would be wise of me to do style components, even if it takes more time.